api/types/container/host_config.go

```diff
@@ -405,6 +405,7 @@ type HostConfig struct {
 	CapDrop         strslice.StrSlice // List of kernel capabilities to remove from the container
 	Capabilities    []string          `json:"Capabilities"` // List of kernel capabilities to be available for container (this overrides the default set)
 	CgroupnsMode    CgroupnsMode      // Cgroup namespace mode to use for the container
+	Devices         []DeviceMapping   `json:",omitempty"` // List of the devices to map to the container
```

Comment:
This is already defined in https://github.com/moby/moby/blob/a90137825c8f2385e2c7eafd82f060c0b8f60059/api/types/container/host_config.go#L361

Which is embedded in the `HostConfig` struct at https://github.com/moby/moby/blob/a90137825c8f2385e2c7eafd82f060c0b8f60059/api/types/container/host_config.go#L434-L435